astropy/units/tests/test_quantity.py

```diff
@@ -1132,7 +1131,7 @@ def test_insert():
     assert q2.unit is u.m
     assert q2.dtype.kind == 'f'
 
-    if NUMPY_VERSION >= version.LooseVersion('1.8.0'):
+    if minversion(np, '1.8.0'):
```

Comment:
I guess I meant: `if not NUMPY_LT_1_8` (but no big deal obviously).  I mostly ask since I think later grepping on `NUMPY_LT` will make for easier updating.